crates/viewer/re_view_spatial/src/transform_cache.rs

```diff
@@ -1705,16 +1874,161 @@ mod tests {
+            let transforms = transforms_per_timeline
+                .entity_transforms(&EntityPath::from("my_entity"))
+                .unwrap();
+
+            // Pose for instances poses and non-boxes are unchanged over time.
+            for t in 1..=4 {
+                for archetype in [
+                    archetypes::InstancePoses3D::name(),
+                    "made_up_archetype".into(),
+                ] {
+                    assert_eq!(
+                        transforms
+                            .latest_at_instance_poses(&LatestAtQuery::new(timeline, t), archetype),
+                        &[
+                            glam::Affine3A::from_translation(glam::Vec3::new(1.0, 2.0, 3.0)),
+                            glam::Affine3A::from_translation(glam::Vec3::new(4.0, 5.0, 6.0)),
+                            glam::Affine3A::from_translation(glam::Vec3::new(7.0, 8.0, 9.0)),
+                        ]
+                    );
+                }
+            }
+
+            // Poses for boxes change over time.
+            // T1
+            assert_eq!(
+                transforms.latest_at_instance_poses(
+                    &LatestAtQuery::new(timeline, 1),
+                    archetypes::Boxes3D::name()
+                ),
+                // All from `InstancePoses3D`
+                &[
+                    glam::Affine3A::from_translation(glam::Vec3::new(1.0, 2.0, 3.0)),
+                    glam::Affine3A::from_translation(glam::Vec3::new(4.0, 5.0, 6.0)),
+                    glam::Affine3A::from_translation(glam::Vec3::new(7.0, 8.0, 9.0)),
+                ]
+            );
+
+            // T2
+            assert_eq!(
+                transforms.latest_at_instance_poses(
+                    &LatestAtQuery::new(timeline, 2),
+                    archetypes::Boxes3D::name()
+                ),
+                // All from `InstancePoses3D` combined with box centers.
+                &[
+                    glam::Affine3A::from_translation(glam::Vec3::new(11.0, 2.0, 3.0)),
+                    glam::Affine3A::from_translation(glam::Vec3::new(4.0, 105.0, 6.0)),
+                    glam::Affine3A::from_translation(glam::Vec3::new(7.0, 108.0, 9.0)), // Affected by the last box center which is still splatted.
+                ]
+            );
+
+            // T3.
+            let query_result = transforms.latest_at_instance_poses(
+                &LatestAtQuery::new(timeline, 3),
+                archetypes::Boxes3D::name(),
+            );
+
+            // More readable sanity check on translations which aren't affected by the rotation.
+            assert_eq!(
+                query_result[0].translation,
+                glam::Vec3A::new(11.0, 2.0, 3.0)
+            );
+            // Since rotation isn't 100% accurate, we need to check for equality with a small tolerance.
+            let eps = 0.000001;
+            // Rotation on the first box affects all insteances since it's splatted.
```

Comment:
```suggestion
            // Rotation on the first box affects all instances since it's splatted.
```
Why is our typo checker not finding these 🤔?